app/views/shared/_meta_tag.html

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
add **title, viewport and initial scale** meta tags also

*plus*: favicon for different resolution